app/celery/scheduled_tasks.py

```diff
@@ -189,3 +199,140 @@ def check_templated_letter_state():
+    try:
+        comp_and_pen_messages: list = _get_dynamodb_comp_pen_messages(table, messages_per_min)
+    except Exception as e:
+        current_app.logger.error(
+            'Exception trying to scan dynamodb table for send_scheduled_comp_and_pen_sms exception_type: %s - '
+            'exception_message: %s', type(e), e
+        )
+        return
+
+    current_app.logger.debug('send_scheduled_comp_and_pen_sms list of items from dynamodb: %s', comp_and_pen_messages)
+
+    # stop if there are no messages
+    if comp_and_pen_messages is None or len(comp_and_pen_messages) < 1:
+        current_app.logger.info(
+            'No Comp and Pen messages to send via send_scheduled_comp_and_pen_sms task. Exiting task.')
+        return
+
+    try:
+        service: Service = dao_fetch_service_by_id(service_id)
+        template: Template = dao_get_template_by_id(template_id)
+    except NoResultFound as e:
+        current_app.logger.error(
+            'No results found in task send_scheduled_comp_and_pen_sms attempting to lookup service or template. Exiting'
+            ' - exception: %s', e)
+        return
+    except Exception as e:
+        current_app.logger.critical(
+            'Error in task send_scheduled_comp_and_pen_sms attempting to lookup service or template Exiting - '
+            'exception: %s', e)
+        return
+
+    # send messages and update entries in dynamodb table
+    for item in comp_and_pen_messages:
+        current_app.logger.info(
+            'sending - item from dynamodb - vaprofile_id: %s | participant_id: %s | payment_id: %s',
+            item.get('vaprofile_id'), item.get('participant_id'), item.get('payment_id')
+        )
+
+        try:
+            # call generic method to send messages
+            send_notification_bypass_route(
+                service=service,
+                template=template,
+                notification_type=SMS_TYPE,
+                personalisation={'paymentAmount': int(item.get('paymentAmount'))},
+                sms_sender_id=service.get_default_sms_sender_id(),
+                recipient_item={
+                    'id_type': IdentifierType.VA_PROFILE_ID.value,
+                    'id_value': str(item.get('vaprofile_id'))
+                },
+            )
+        except Exception as e:
+            current_app.logger.critical(
+                'Error attempting to send Comp and Pen notification with send_scheduled_comp_and_pen_sms | item from '
+                'dynamodb - vaprofile_id: %s | participant_id: %s | payment_id: %s | exception_type: %s - '
+                'exception: %s', item.get('vaprofile_id'), item.get('participant_id'), item.get('payment_id'), type(e),
+                e
+            )
+
+        current_app.logger.info(
+            'sent to queue, updating - item from dynamodb - vaprofile_id: %s | participant_id: %s | payment_id: %s ',
+            item.get('vaprofile_id'), item.get('participant_id'), item.get('payment_id')
+        )
```

Comment:
```suggestion
        else:
            current_app.logger.info(
                'sent to queue, updating - item from dynamodb - vaprofile_id: %s | participant_id: %s | payment_id: %s ',
                item.get('vaprofile_id'), item.get('participant_id'), item.get('payment_id')
            )
```
This seems like it should be in an `else` in the event the exception above was thrown.